### PR TITLE
Update rubygem-signet to 0.11.0

### PR DESCRIPTION
**Pull request by evgeni**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

We use signet 0.11.0 in our CI, so let's just also use it for prod setups? :)

https://ci.theforeman.org/job/test_develop/4010/database=postgresql,ruby=2.5,slave=fast/consoleFull

Also re-generated the spec file based on the latest SCL template

### CODE REVIEW

**Comment by evgeni** [comment]
reposclosure failure is because #3466 isn't merged yet.

File: packages/foreman/rubygem-signet/rubygem-signet.spec [hunk 1]
```diff
@@ -1,85 +1,101 @@
...
+Requires: %{?scl_prefix}rubygem(jwt) >= 1.5
+Requires: %{?scl_prefix}rubygem(jwt) < 3.0
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
```
Comment (ekohl) [inline packages/foreman/rubygem-signet/rubygem-signet.spec]: I've been keeping these in until now, but I do wonder if we should start dropping these

Comment (evgeni) [reply packages/foreman/rubygem-signet/rubygem-signet.spec]: I'd nuke them once and for all, the `ruby193` SCL is so obsolete :)

**Comment by evgeni** [comment]
build: http://koji.katello.org/koji/taskinfo?taskID=174905

**Pull request merged**